tr.gov.uyap.Editor.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>tr.gov.uyap.Editor</id>
+
+  <name>UYAP Editor</name>
+  <summary>UYAP Document Editor is an application to display and create UDF documents in UYAP
+    system.</summary>
+
+  <metadata_license>CC-BY-4.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Is it redistributable? If not, the manifest should be changed to use `extra-data`

Reply:
Yes it is redistributable